azkaban-common/src/main/java/azkaban/project/AzkabanProjectLoader.java

```diff
@@ -49,6 +48,7 @@
 class AzkabanProjectLoader {
 
   private static final Logger log = LoggerFactory.getLogger(AzkabanProjectLoader.class);
+  private static final String DIRECTORY_FLOW_REPORT_KEY = "Directory Flow";
```

Comment:
Is this part of the user-facing contract? Where is it used?

Reply:
I don't think it's a user-facing contract. It is used for displaying the error/warn messages to users when there is something wrong in uploading projects. It used to be: `public static final String DEFAULT_VALIDATOR_KEY = "Directory Flow";`. I just want to keep the same UI display.

Reply:
Got it. Thanks